src/Fabulous.AST.Tests/UnitTest1.fs

```diff
@@ -0,0 +1,22 @@
+module Fabulous.AST.Tests
+
+open NUnit.Framework
+
+open Fabulous.AST
+open type Fabulous.AST.Node
+
+[<Test>]
+let CanCompileBasicTree () =
+    let tree =
+        ModuleOrNamespace(
+            Range("test.fs", Position(1, 2), Position(3, 4))
+        )
+        
+    let rootNode = Tree.compile tree :?> Fantomas.Core.SyntaxOak.ModuleOrNamespaceNode
+    
+    Assert.NotNull(rootNode.Range)
```

Comment:
I highly encourage you to write tests that assert either the `Oak` or the formatted code.
That way, when upgrading to a newer version of Fantomas you will know what still works.